Library/Formula/dependency-check.rb

```diff
@@ -0,0 +1,41 @@
+class DependencyCheck < Formula
+  desc "OWASP Dependency Check"
+  homepage "https://www.owasp.org/index.php/OWASP_Dependency_Check"
+  url "https://dl.bintray.com/jeremy-long/owasp/"\
```

Comment:
The style guide says that lines need to be 80 characters or fewer - the URL continues on the following line.